backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -4,27 +4,9 @@ table {
   border-spacing: 0;
 
   th, td {
-    border-right: 1px solid $color-border;
-    border-bottom: 1px solid $color-border;
-    vertical-align: middle;
+    vertical-align: top;
```

Comment:
@graygilmore @Mandily I know why the vertical alignment for single line rows is off. The action buttons occupy more space then the text and thats why the row is too high. 

![monosnap 2017-07-19 21-09-55](https://user-images.githubusercontent.com/42868/28384932-a7abd826-6cc6-11e7-9998-c936311d41c5.jpg)

Solution would be to get rid of the round background. 

![option types - products 2017-07-19 21-13-06](https://user-images.githubusercontent.com/42868/28385079-1550722e-6cc7-11e7-948f-7022a13856b3.jpg)

We should handle this problem in separate PR. 
WDYT about this approach?